control-plane/msp-operator/src/main.rs

```diff
@@ -859,6 +879,20 @@ async fn pool_controller(args: ArgMatches<'_>) -> anyhow::Result<()> {
     Ok(())
 }
 
+/// Generate the mayastor pool CRD file allowing users to register them with kubernetes
+/// before the pool operator starts running.
+/// Can also be used to unregister the CRDs on uninstall.
+fn write_msp_crd(name: Option<&str>) -> anyhow::Result<()> {
```

Comment:
We don't need the crd in yaml to delete the crd definition; a simple 

`k  delete crd mayastorpools.openebs.io` 

is sufficient, but I see what you are trying to say here :+1:

Reply:
Right, good point!